### PR TITLE
Added notebook for: Custom training image classification model for online prediction with unmanaged dataset

### PR DESCRIPTION
**Pull request by ivanmkc**
Added a notebook demonstrating 'Custom training image classification model for online prediction with unmanaged dataset'

## TODOs
- [x] Fix the link to Colab and Github.
- [x] custom-training-online-prediction.ipynb

### CODE REVIEW

**Comment by thedriftofwords** [comment]
@ivanmkc I provided a partial review of the online prediction notebook - however, there are more substantial changes that need to be made, so after we get those comments addressed, I'll do a cleanup PR to handle the rest of it.

File: .cloud-build/ExecuteChangedNotebooks.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def run_changed_notebooks(
     notebooks = notebooks.decode("utf-8").split("\n") + test_notebooks
     notebooks = [notebook for notebook in notebooks if notebook.endswith(".ipynb")]
     notebooks = [notebook for notebook in notebooks if len(notebook) > 0]
+    notebooks = [notebook for notebook in notebooks if Path(notebook).exists()]
```
Comment (ivanmkc) [inline .cloud-build/ExecuteChangedNotebooks.py:75]: Added check for notebooks that exist

**Review by thedriftofwords — APPROVED** [review]
Note: Next, I'll make a follow-up PR to add further changes myself, but for now this is merge-able. Thanks!

**Pull request merged**